### ISSUE TITLE
Google Maps SDK for IOS 3.0.3 was released

### ISSUE THREAD

**Issue by battika** [mapsplugin/cordova-plugin-googlemaps#2594]
Hello Masashi,

Not sure if you intend to update the plugin, just wanted to let you know that IOS SDK version 3.0.3 was released.

Here is the relate podspec:
https://github.com/CocoaPods/Specs/blob/master/Specs/a/d/d/GoogleMaps/3.0.3/GoogleMaps.podspec.json

Changelog since 2.7.0 mentions the following fixes:
```
Version 3.0.3 - February 2019
==========================
Resolved Issues:
  - Fixes the occasional rendering bug when the using polygon overlay feature.

Version 3.0.0 - January 2019
==========================
Note:
  - Support for iOS 8.0 has been dropped. The new minimum is iOS 9.0.

Known Issues:
  - There is an occasional rendering bug when the using polygon overlay feature.
```

**Comment by wf9a5m75** [comment]
As far as I tested, cocoapod can not find out v3.0.3 for some reasons (Cordova fails installation).
I specified as `~> 3.0.2`.

**Comment by wf9a5m75** [comment]
I think cocoapod problem. Try to upgrade using cocoapod command.

**Comment by wf9a5m75** [comment]
cordova-plugin-googlemaps has been released, which uses Google Maps SDK for iOS v3.1.0
